lib/ui/screens/contact_list_screen.dart

```diff
@@ -95,6 +99,15 @@ class _ContactListScreenState extends State<ContactListScreen> {
     );
   }
 
+  _onChanged(String value){
+    setState(() {
+      List<Contact> contactList = ContactsRepository.getAllContacts()!;
```

Comment:
ContactsRepository.getAllContacts() should always return List<Contact>, not optional value.